packages/dd-trace/src/log.js

```diff
@@ -7,9 +7,22 @@ const _default = {
   error: err => console.error(err) /* eslint-disable-line no-console */
 }
 
+const _logLevels = ['trace', 'debug', 'info', 'warn', 'error']
+
 let _logger
 let _enabled
 let _deprecate
+let _logLevel
+
+const _isLogLevelEnabled = (level) => {
+  return !_logLevel || _logLevels.indexOf(level) >= _logLevels.indexOf(_logLevel)
+}
+
+const _setLogLevel = (logLevel) => {
+  if (logLevel && typeof logLevel === 'string') {
```

Comment:
Should this be validated? Currently, setting this to an incorrect value could result in everything being logged.

Reply:
the current behavior is that if `debug` mode is not enabled, all of the  tracer's `log` calls are `noops`, and if it is enabled, there's no restriction. So the goal here was to match that behavior by just allowing an incorrect value to permit all the same log levels as if this setting wasn't set at all. I've tried to make that a bit more explicit by adding a `_defaultLogLevel` constant that is set to debug, and in the case of an invalid value (basically any value, string or otherwise, that isn't `error` or `debug`), it is set to the `_defaultLogLevel`